launch/loc.launch

```diff
@@ -0,0 +1,62 @@
+<launch>
+  <arg name="model" default="$(find mrover)/rover_description/rover.urdf" />
+  <arg name="gui" default="false" />
+  <param name="robot_description" command="xacro $(find mrover)/rover_description/rover.xacro" />
+  <arg name="rvizconfig" default="$(find mrover)/loc.rviz" />
+
+  <node if="$(arg gui)" name="joint_state_publisher" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui" />
+  <node unless="$(arg gui)" name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
+  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
+  <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />
+
+  <include file="$(find gazebo_ros)/launch/empty_world.launch">
+    <env name="GAZEBO_MODEL_PATH" value="$(find mrover)/env_description" />
+    <arg name="world_name" value="$(find mrover)/env_description/world.world" />
+    <arg name="paused" value="false" />
+    <arg name="use_sim_time" value="true" />
+    <arg name="gui" value="true" />
+    <arg name="headless" value="false" />
+    <arg name="debug" value="false" />
+    <arg name="verbose" default="false" />
+    <arg name="gui_required" value="true" />
+  </include>
+
+  <node name="spawn_model" pkg="gazebo_ros" type="spawn_model" output="screen" args="
+    -urdf -param robot_description
+    -model rover" />
+
+  <!-- Navigation -->
+  <!-- This publishes the transforms of the course waypoints -->
+  <node name="debug_course_publisher" pkg="mrover" type="debug_course_publisher.py" />
+
+  <!--
+    ============
+    Localization
+    ============
+    Frames in the free tree:
+    odom: The topmost global frame
```

Comment:
what actually publishes /odom? I only see odometry/filtered in this launch file?